src/main/java/uk/gov/companieshouse/ocr/api/image/extracttext/ImageOcrService.java

```diff
@@ -0,0 +1,135 @@
+package uk.gov.companieshouse.ocr.api.image.extracttext;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.concurrent.CompletableFuture;
+
+import javax.imageio.ImageIO;
+import javax.imageio.ImageReader;
+import javax.imageio.stream.ImageInputStream;
+
+import org.apache.commons.lang.time.StopWatch;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import net.sourceforge.tess4j.ITessAPI;
+import net.sourceforge.tess4j.ITessAPI.TessBaseAPI;
+import net.sourceforge.tess4j.ITessAPI.TessPageIteratorLevel;
+import net.sourceforge.tess4j.TessAPI;
+import net.sourceforge.tess4j.TesseractException;
+import net.sourceforge.tess4j.util.ImageIOHelper;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.ocr.api.OcrApiApplication;
+import uk.gov.companieshouse.ocr.api.tesseract.TesseractConstants;
+
+@Service
+public class ImageOcrService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(OcrApiApplication.APPLICATION_NAME_SPACE);
+
+    @Async
+    public CompletableFuture<TextConversionResult> extractTextFromImage(MultipartFile file, String externalReferenceId, StopWatch timeOnQueueStopWatch) throws IOException, TesseractException {
```

Comment:
Not sure I like the idea of passing in a StopWatch.

Reply:
Can you expand on this.

The alternative would be to add a long for the current time and then set the Queue time to that in the service code. I prefer the above since it seems simpler to me